pandas/core/indexes/period.py

```diff
@@ -191,9 +205,11 @@ class PeriodIndex(DatelikeOps, DatetimeIndexOpsMixin, Int64Index):
 
     Examples
     --------
-    >>> idx = PeriodIndex(year=year_arr, quarter=q_arr)
+    >>> idx = pd.PeriodIndex(year=[2000, 2001, 2002], quarter=[1, 2, 1])
 
-    >>> idx2 = PeriodIndex(start='2000', end='2010', freq='A')
+    >>> idx2 = pd.PeriodIndex(start='2000', end='2010', freq='A')
+
+    >>> idx3 = pd.PeriodIndex(year=[2000, 2001], month=[1, 2], second=[1, 2], freq='S')
 
     See Also
     ---------
```

Comment:
Can you add `Interval` and `IntervalIndex` to the See Also section? Seems relevant as they can be used for more generic intervals of time in that they don't require a specific frequency.